scripts/translationsToMultipleFiles.js

```diff
@@ -0,0 +1,72 @@
+const fs = require('fs');
+const yaml = require('js-yaml');
+
+// Importing CommonJS sucks, so we'll copy some minor things
+const fileMiddleName = '-single-file-from-'
+const encoding = 'utf8'
+const folderPrefix = 'folder_'
+const yamlName = (language) => `${language}.yaml`
+const yamlDumpOptions = {
+    'flowLevel': -1,
+    'lineWidth': -1,
+    'noRefs': true
+}
+
+function makeMultipleFiles(singleTranslationFileName, translationsFolder='../translations') {
+    const translationYaml = loadYaml(singleTranslationFileName)
+    yamlDictionariesFrom(translationYaml, translationsFolder, translationLanguage(singleTranslationFileName)).forEach(({ directory, content }) => {
+        fs.writeFileSync(directory, yaml.safeDump(content, yamlDumpOptions), encoding)
+    })
+}
+
+const translationLanguage = (singleTranslationFileName) => singleTranslationFileName.split(fileMiddleName)[0]
+
+function loadYaml(singleTranslationFileName) {
+    const fileContents = fs.readFileSync(yamlName(singleTranslationFileName), encoding)
+    return yaml.safeLoad(fileContents)
+}
+
+function yamlDictionariesFrom(translationYaml, baseDirectory, language) {
+    let yamlDictionaries = []
+    const singleFileTranslationYaml = {... translationYaml}
+    Object.keys(translationYaml)
+    .filter(isFolder)
+    .forEach(k => {
+        yamlDictionaries = yamlDictionaries.concat(
+            yamlDictionariesFrom(
+                translationYaml[k],
+                `${baseDirectory}/${folderName(k)}`,
+                language
+            )
+        )
+        delete singleFileTranslationYaml[k]
+    })
+
+    const yamlDictionary = yamlDictionaryFrom(singleFileTranslationYaml, baseDirectory, language)
+
+    // Some folders do not have their own translation file.
+    if(!isEmpty(yamlDictionary.content)) yamlDictionaries.push(yamlDictionary)
+
+    return yamlDictionaries
+}
+
+function isEmpty(obj) { 
+    for (var x in obj) { return false; }
+    return true;
+ }
```

Comment:
```suggestion
const isEmpty = (obj) => Object.keys(obj).length === 0;
```

Reply:
La razón por la cual lo había hecho así es que había leído que no lo bancaban todos los browsers (peeero claramente eso era re viejo y ya sucede). Y pensé que funcionaba para las [propiedades no enumerables](https://www.tutorialspoint.com/what-is-non-enumerable-property-in-javascript-and-how-can-it-be-created), pero no funciona tampoco jajaja. Se puede usar `getOwnPropertyNames` pero meh, ni me interesa.

Commiteo porque me gusta más esa solución